packages/slice-machine/components/Changelog/versionDetails/UpdateCommandBox.tsx

```diff
@@ -39,6 +49,18 @@ export const UpdateCommandBox: React.FC<UpdateCommandBoxProps> = ({
       >
         How to install
       </Text>
+      {isOnlyAdapterUpdate && isLatestSliceMachineVersion && (
+        <Text
+          sx={{
+            fontSize: "12px",
+            fontWeight: 500,
+            lineHeight: "16px",
+          }}
+        >
+          It looks like you are using an outdated version of the adapter. Run
+          this command to update your adapter:
```

Comment:
Could we get Sam Littlefair's (@samlfair) approval on this message if he hasn't already reviewed it?

Reply:
Yep, this message is coming from him 🙂